Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -987,6 +995,11 @@ void CPacketHandler::Packet_PlayerList(NetBitStreamInterface& bitStream)
             // Call the onClientPlayerJoin event
             CLuaArguments Arguments;
             pPlayer->CallEvent("onClientPlayerJoin", Arguments, true);
+
+            if (g_pCore->GetNetwork()->GetServerBitStreamVersion() >= 0x06D)
+                g_pCore->GetDiscordManager()->SetState(SString("%i/%i Players", g_pClientGame->GetPlayerManager()->Count(), g_pClientGame->GetServerInfo()->GetMaxPlayers()), [](EDiscordRes) {});
+            else
+                g_pCore->GetDiscordManager()->SetState(SString("%i Players", g_pClientGame->GetPlayerManager()->Count()), [](EDiscordRes) {});
```

Comment:
This (and other places) needs to say `1 Player` if there is only 1 player. 

Are you able to do something about code duplication?

Reply:
Resolved